.circleci/config.yml

```diff
@@ -76,11 +76,14 @@ references:
   deploy_cluster: &deploy_cluster
    run:
      name: Deploy the job on the kubernetes cluster
+     no_output_timeout: 20m  # by default, the job terminates after 20min of no output.
```

Comment:
pretty nice! But I see that the job ran for ~38 mins.

Reply:
I think (guess) this is fine because the following line prints `.` per `$CHECK_SPEEP` seconds, so there won't be a case where there's no output for `20m`:

```
       while [ $i -lt $MAX_CHECKS ]; do ((i++)); if kubectl get jobs $job_name -o jsonpath='Failed:{.status.failed}' | grep "Failed:1"; then status_code=1 && break; elif kubectl get jobs $job_name -o jsonpath='Succeeded:{.status.succeeded}' | grep "Succeeded:1" ; then status_code=0 && break; else printf "."; fi; sleep $CHECK_SPEEP; done && \
```